src/common/components/CopyButton.tsx

```diff
@@ -12,3 +14,71 @@ export const CopyButton = ({
     </ClipboardRoot>
   );
 };
+
+export const CopyButtonNew = ({
+  initialValue,
+  ...rest
+}: { initialValue: string } & IconButtonProps) => {
+  const { height, width, h, w, color } = rest;
+  const heightValue = h || height;
+  const widthValue = w || width;
```

Comment:
can we just pass the dimensions to child components? adding logic here could cause problems like if `h` is set to explicit 0.

Reply:
done